esmvaltool/preprocessor/_multimodel.py

```diff
@@ -164,133 +177,73 @@ def _get_overlap(cubes):
     """
     all_times = []
     for cube in cubes:
-        # monthly data ONLY
-        # converts all to 365-days calendars
-        time_units = cube.coord('time').units
-        bnd1 = float(cube.coord('time').points[0])
-        bnd2 = float(cube.coord('time').points[-1])
-        if time_units.calendar is not None:
-            if time_units.calendar == '360_day':
-                bnd1 = int(bnd1 / 360.) * 365.
-                bnd2 = (int(bnd2 / 360.) + 1) * 365.
-            else:
-                bnd1 = int(bnd1 / 365.) * 365.
-                bnd2 = (int(bnd2 / 365.) + 1) * 365.
-        else:
-            logger.debug("No calendar type: assuming 365-day")
-            bnd1 = int(bnd1 / 365.) * 365.
-            bnd2 = (int(bnd2 / 365.) + 1) * 365.
+        bnd1 = _datetime_to_int_days(cube)[0]
+        bnd2 = _datetime_to_int_days(cube)[-1]
```

Comment:
avoid calling `_datetime_to_int_days()` twice, e.g. like this:
```
bnds = _datetime_to_int_days(cube)
bnd1, bnd2 = bnds[0], bnds[-1]
```

Reply:
Renaming `bnd1` and `bnd2` to `start` and `end` (or `stop`) would make this more readable.

Reply:
done!